src/seraphis_main/tx_component_types_legacy.cpp

```diff
@@ -77,15 +89,15 @@ void append_to_transcript(const LegacyRingSignatureV4 &container, SpTranscriptBu
 //-------------------------------------------------------------------------------------------------------------------
 std::size_t legacy_ring_signature_v4_size_bytes(const std::size_t num_ring_members)
 {
-    return clsag_size_bytes(num_ring_members) + num_ring_members * 8;
+    return clsag_size_bytes(num_ring_members) + 8 + num_ring_members * 8;
```

Comment:
As is, shouldn't it be this since each member now gets a `ledger_indexing_amount` too?

```suggestion
    return clsag_size_bytes(num_ring_members) + num_ring_members * 16;
```